lib/_http_incoming.js

```diff
@@ -21,6 +30,7 @@ function readStop(socket) {
 }
 exports.readStop = readStop;
 
+
```

Comment:
This is to address your [earlier](https://github.com/nodejs/node/pull/11403#discussion_r101404130) review comment.